### PR TITLE
Fix: Creating or removing node URL aliases results in a fatal error

### PR DESCRIPTION
**Pull request by peterkeung**
On a legacy bridge install, creating or removing a node URL alias (via content/urlalias/<node_id>) results in this error:

`Unexpected error, the message was : Argument 1 passed to eZ\Bundle\EzPublishLegacyBundle\Cache\SwitchableHttpCachePurger::purge() must be of the type array, string given, called in ezpublish_legacy/kernel/private/classes/ezpevent.php on line 138 in vendor/netgen/ibexa-legacy-bridge/bundle/Cache/SwitchableHttpCachePurger.php on line 38`

In short, the first parameter in the array or parameters sent to the "content/cache" event needs to be an array itself:

`ezpEvent::getInstance()->notify( 'content/cache', array( array( $NodeID ) ) );`

This is because SwitchableHttpCachePurger::purge() requires the array: https://github.com/netgen/ibexa-legacy-bridge/blob/3.x/bundle/Cache/SwitchableHttpCachePurger.php#L38

### CODE REVIEW

**Comment by emodric** [comment]
Thanks @peterkeung !
